xbmc/windowing/gbm/VTUtils.cpp

```diff
@@ -23,7 +23,21 @@ using namespace KODI::WINDOWING::GBM;
 
 bool CVTUtils::OpenTTY()
 {
-  m_ttyDevice = ttyname(STDIN_FILENO);
+  const char* ttyDevice = ttyname(STDIN_FILENO);
```

Comment:
Can we change this to `ttyname_r` while we are at it?

Reply:
Does it really add any benefit?

Reply:
It's the reentrant version.
I usually prefer them in utils code, just to skip thinking about threading issues.

Reply:
Updated